### DISCUSSION TITLE
Phone won't connect

### DISCUSSION THREAD

**Discussion by MONKE121** [vicwomg/pikaraoke#281] [Q&A]
Please help I installed PiKaraoke but ip address won't reach on phone or any other devices. It wont connect thru other even with the same wifi. I don't have any knowledge when it comes to network.

**Comment by vicwomg** [comment]
What IP address is shown on the screen? Are you sure that the pi is connected to your network?

**Reply by MONKE121** [reply]
Yes I am use pi is connected as I can download songs only on computer it works well . The only problem is phone won't connect to the QR Code or the IP address it says"ER_ADDRESS_UNREACHEABLE.

the IP Adress 192.168.1.20:5050

**Reply by MONKE121** [reply]
A friend of mine who even an IT student can't figure out the problem. My computer is wired thru a hub and another wire thru the main router.

So my computer is connect via LAN 

Phone is connected via WLAN 

My question is pikaraoke can connect it? Sorry im not good at techs, I just found out the Pikaraoke thru Reddit.

**Reply by vicwomg** [reply]
How is the pi connected? Is it connected to WLAN or LAN? 

It seems to me like your WLAN cannot reach your LAN. Do you have access to the router? The router might be putting WLAN on a different IP range and it's not accessible to the LAN clients. Do you have access to the router or a network administrator that you can ask? That's my best guess.

It's also possible your router has some restrictions on the port 5050. You could also try running pikaraoke on a different port and see if it works better. Use the `-p 5555` option.

**Reply by MONKE121** [reply]
Oh sorry I forgot to told yo I didn't have a raspberry PI. I am using PC windows 10 and a xiaomi phone.

**Reply by MONKE121** [reply]
Pikaraoke is installed on my PC windows 10 connected thru LAN


Yes I have access to the router, I can open the network administration but I don't know how to configure it. 


I will try 5555 option. Thank you

**Reply by vicwomg** [reply]
If you can find the IP of the Xiaomi phone in Android settings, that should make it pretty clear if it's on a separate IP range. It needs to be on the same subnet as your pikaraoke host `192.168.1.XXX`, if it's not in that range it wont be able to reach your server. This is a networking configuration issue and I cannot help you troubleshoot your router, unfortunately.

**Reply by jramboz** [reply]
It could be that the Windows firewall is blocking the incoming connection. I’m not sure where the setting for that is exactly. It should be somewhere under the security settings, I think.

**Reply by MONKE121** [reply]
I downloaded an app that shows all IP around on my home, my phone IP is 192.168.1.XX .

I can't have internet if on my "Internet Protocol Version 4 (TCP/IPv4) Properties " if I turn on "Obtain an IP autumatically". and I check the network administration, my Computer won't show up in the LAN. 

![Capture](https://github.com/vicwomg/pikaraoke/assets/112748093/a63a9a27-3fb2-421a-8177-e6b3d6f26fdd)

I hope somebody can help me.

**Reply by vicwomg** [reply]
As @jramboz  said, I think your windows firewall is blocking incoming connections. See this guide on how to allow apps through your firewall.

https://pureinfotech.com/allow-apps-firewall-windows-10/

I don't run Windows, so I can't help much more than that.

**Reply by MONKE121** [reply]
I done this method I allowed pikaraoke as I ping on my phone from a net analyser app. Phone can't detect my computer. Only those on WLAN. 

another question, is there a way to install the pikaraoke on SmartTV? 

Thank you

**Reply by MONKE121** [reply]
I even reset to default the Firewall

**Reply by Hartebee5t** [reply]
@MONKE121 
Does your phone connect successfully to your computer (via your LAN, wireless or hardwired) using any other applications?

**Reply by MONKE121** [reply]
> @MONKE121 
> Does your phone connect successfully to your computer (via your LAN, wireless or hardwired) using any other applications?
> 
> 

Only when using USB tethering it can connect. The only wireless it can connect with TV and Phone is via Youtube casting. I was hoping to use the PiKaraoke on PC then cast it on TV via Wireless or HDMI then people can scan the QR code with IP address to the PC to be able select songs on PiKaraoke.

**Reply by MONKE121** [reply]
I am wondering if it is possible to change the IP address of Pikaraoke to match the IP address of my router. I have observed that my PC can successfully detect the router's IP address, but not the phone, and similarly, my phone can detect the router but not the PC.

I appreciate any guidance or assistance you can provide in resolving this matter. Please let me know if there are steps I can take to achieve better connectivity between my devices. Thank you for your time and assistance.
